modules/ROOT/pages/7.5-release-notes.adoc

```diff
@@ -91,7 +91,24 @@ For information on the **Markdown** plugin, see xref:markdown.adoc[Markdown].
 
 The {productname} {release-version} release includes an accompanying release of the **Comments** premium plugin.
 
-**Comments** Premium plugin includes the following fixes.
+**Comments** Premium plugin includes the following additions and fixes.
+
+==== Comments with Mentions
+// EPIC-111
+
+The **Comments** plugin now supports mentions in comments.
+
+Previously, the **Comments** plugin allowed users to add comments to content in the editor. However, users could not mention other users in the comments sidebar, limiting the ability to notify specific users or direct comments to them.
+
+In {productname} {release-version}, the **Comments** plugin has been enhanced to support xref:mentions.adoc[Mentions] in comments. Users can now mention other users in comments by typing the `+@+` symbol followed by the user's name. This feature enables users to notify specific individuals, direct comments to them, or engage in conversations with other users directly within the editor.
```

Comment:
```suggestion
In {productname} {release-version}, the **Comments** plugin has been enhanced to support xref:mentions.adoc[Mentions] in comments. Users can now mention other users in comments by typing the `+@+` symbol followed by the user's name.
```
Last sentence is more dependent upon the integration, and users can already engage in conversations with comments